go-controller/pkg/cni/cni.go

```diff
@@ -115,6 +118,37 @@ func (pr *PodRequest) cmdAdd() *PodResult {
 		return nil
 	}
 
+	ipAddressMask := strings.Split(ipAddress, "/")
+	if len(ipAddressMask) != 2 {
+		logrus.Errorf("Error to get ip address")
```

Comment:
Maybe the error should say that the IP address is not in the valid format and should print the `ipaddress` in the debug message